plugins/polio/tests/test_vaccine_authorizations.py

```diff
@@ -658,10 +659,8 @@ def test_expired_vaccine_authorizations_email_alert(self):
         # test the function itself to check if the content is correct
 
         response = expired_vaccine_authorizations_email_alert(vaccine_auths, mailing_list)
-
-        page_url = f"example.com/dashboard/polio/vaccinemodule/nopv2authorisation/accountId/{self.team.project.account.id}/order/-current_expiration_date/pageSize/20/page/1"
+        page_url = f"{Site.objects.get_current()}/dashboard/polio/vaccinemodule/nopv2authorisation/accountId/{self.team.project.account.id}/order/-current_expiration_date/pageSize/20/page/1"
```

Comment:
I'd rather use `settings.DNS_DOMAIN` since we decided to not rely on `current_site.domain` in #1378.